packages/formatters/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "messageformat-formatters",
+  "version": "0.1.0",
+  "description": "Formatters for messageformat",
+  "keywords": [
+    "icu",
+    "messageformat",
+    "formatter"
+  ],
+  "author": "Eemeli Aro <eemeli@gmail.com>",
+  "license": "MIT",
+  "homepage": "https://messageformat.github.io/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/messageformat/messageformat.git",
+    "directory": "packages/formatters"
+  },
+  "main": "index.js",
+  "files": [
+    "index.js",
+    "lib/"
+  ],
+  "dependencies": {
+    "cldr-compact-number": "^0.4.0"
```

Comment:
<img width="469" alt="Screen Shot 2019-11-24 at 9 49 00 PM" src="https://user-images.githubusercontent.com/7374640/69513198-40393c00-0f04-11ea-94f1-817abd0ebd73.png">

Note I include en locale information by default, so it is usable out of the box.